README.md

```diff
@@ -156,7 +156,7 @@ npm install --save-dev ts-essentials
 
 ### Type checkers
 
-- [`Exact<Type, Shape>`](/lib/exact) - Returns `Type` when type `Type` and `Shape` are identical. Otherwise returns
+- [`IsExact<Type, Shape>`](/lib/is-exact) - Returns `Type` when type `Type` and `Shape` are identical. Otherwise returns
```

Comment:
nit: can you place it between `IsAny` and  `IsNever` to keep alphabetical order?